github-actions.cabal

```diff
@@ -82,7 +82,6 @@ test-suite spec
   other-modules:      Language.Github.Actions.WorkflowTest
   build-tool-depends: tasty-discover:tasty-discover ^>=4.2.2
   build-depends:
-    , bytestring          ==0.11.5.3  || ==0.12.1.0
```

Comment:
I would depend on `bytestring ^>=0.11.4.0 || ^>=0.12.0.2` (the first 0.12 series released with ghc, and the 0.11 release that came with GHC 9.6).

No need to go back to older versions, but bytestring is bundled with every GHC and it's not worth doing backflips to avoid the dep.